server/static/partials/student/assignment.dash.html

```diff
@@ -2,11 +2,12 @@
 
   <div class="assign large active" id="1" ng-repeat="assign in assignments">
     <div class="assign-main">
+      <div class="score">{{ ::assign.assignment.score }}<span class="score-total">/{{::assign.assignment.points}}</span><span class="score-pts">PTS</span></div>
       <img class="assign-bg" src="/static/student/images/bg-clock.png">
        <h2 class="assign-title"> {{ ::assign.assignment.display_name }}</h2>
       <div class="assign-deck">
         <p><img src="/static/student/images/icon-clock.png">due {{ ::assign.assignment.due_date | amCalendar }}</p>
-        <p><span class="icon-question">?</span> {{::assign.assignment.points}} points</p>
+        <p><span class="icon-question">?</span> --/{{::assign.assignment.points}} points</p>
```

Comment:
Can you remove this line too. We'll just rely on the bubble for the grade and if it's ungraded the bubble will say --/25